README.md

```diff
@@ -114,12 +114,18 @@ name                  | description                                | default val
 eclair.datadir        | Path to the data directory                 | ~/.eclair
 eclair.headless       | Run eclair without a GUI                   | 
 eclair.printToConsole | Log to stdout (in addition to eclair.log)  |
+logback.configurationFile | The [`logback`](https://logback.qos.ch) configuration file to use |
```

Comment:
I'd rather not mix our own settings and logback settings and instead add a small "logging" section belows, which just says that we use logback for logging and explains how to override the default config as you did below,